.travis.yml

```diff
@@ -13,6 +13,7 @@ env:
   global:
     - DATABASE_URL=postgres://postgres@localhost:5432/herbie_db
     - SECRET_KEY=thisisasecret
+    - MESSAGING_PROVIDER=kafka
```

Comment:
Why do we configure the Message-Provider as environment variable? What is the advantage?

Reply:
I use this environment variable in order to to identify which publisher should i create in MessagePublisher class in this exactly part 
        if settings.MESSAGING_PROVIDER == 'kafka':
            self.messaging_provider = KafkaPublisher()
        elif settings.MESSAGING_PROVIDER == 'google_pub_sub':
            self.messaging_provider = GooglePubSubPublisher()
Can we do it with some other way?

Reply:
I get that part, but my question is why we need to define it as environment variable and not in the settings? The way how it is implemented right now implies that all different event-stream packages are always installed.

Reply:
You are right, I just wanted for anyone that fork the project, to be able to run any different combination(kafka or pubsub) with just one command(docker-compose up) without having to change anything in the setting or in the docker-compose files. I will remove it from docker-compose. Also in travis we actually don't need it we can initialize it before the tests run in the setUp or setUpClass functions of tests